src/target/adiv5.h

```diff
@@ -23,143 +23,165 @@
+#define ADIV5_DP_CTRLSTAT_CDBGRSTREQ   (1u << 26)
 /* Bits 25:24 - Reserved */
 /* Bits 23:12 - TRNCNT */
-#define ADIV5_DP_CTRLSTAT_TRNCNT        (1u << 12)
+#define ADIV5_DP_CTRLSTAT_TRNCNT (1u << 12)
 /* Bits 11:8 - MASKLANE */
 #define ADIV5_DP_CTRLSTAT_MASKLANE
 /* Bits 7:6 - Reserved in JTAG-DP */
-#define ADIV5_DP_CTRLSTAT_WDATAERR	(1u << 7)
-#define ADIV5_DP_CTRLSTAT_READOK	(1u << 6)
-#define ADIV5_DP_CTRLSTAT_STICKYERR	(1u << 5)
-#define ADIV5_DP_CTRLSTAT_STICKYCMP	(1u << 4)
-#define ADIV5_DP_CTRLSTAT_TRNMODE_MASK	(3u << 2)
-#define ADIV5_DP_CTRLSTAT_STICKYORUN	(1u << 1)
-#define ADIV5_DP_CTRLSTAT_ORUNDETECT	(1u << 0)
-
+#define ADIV5_DP_CTRLSTAT_WDATAERR     (1u << 7)
+#define ADIV5_DP_CTRLSTAT_READOK       (1u << 6)
+#define ADIV5_DP_CTRLSTAT_STICKYERR    (1u << 5)
+#define ADIV5_DP_CTRLSTAT_STICKYCMP    (1u << 4)
+#define ADIV5_DP_CTRLSTAT_TRNMODE_MASK (3u << 2)
+#define ADIV5_DP_CTRLSTAT_STICKYORUN   (1u << 1)
+#define ADIV5_DP_CTRLSTAT_ORUNDETECT   (1u << 0)
 
 /* ADIv5 MEM-AP Registers */
-#define ADIV5_AP_CSW		ADIV5_AP_REG(0x00)
-#define ADIV5_AP_TAR		ADIV5_AP_REG(0x04)
+#define ADIV5_AP_CSW ADIV5_AP_REG(0x00)
+#define ADIV5_AP_TAR ADIV5_AP_REG(0x04)
 /* 0x08 - Reserved */
-#define ADIV5_AP_DRW		ADIV5_AP_REG(0x0C)
-#define ADIV5_AP_DB(x)		ADIV5_AP_REG(0x10 + (4*(x)))
+#define ADIV5_AP_DRW   ADIV5_AP_REG(0x0C)
+#define ADIV5_AP_DB(x) ADIV5_AP_REG(0x10 + (4 * (x)))
 /* 0x20:0xF0 - Reserved */
-#define ADIV5_AP_CFG		ADIV5_AP_REG(0xF4)
-#define ADIV5_AP_BASE		ADIV5_AP_REG(0xF8)
-#define ADIV5_AP_IDR		ADIV5_AP_REG(0xFC)
-
-/* Known designers seen in SYSROM-PIDR. Ignore Bit 0 from
- * the designer bits to get JEDEC Ids with bit 7 ignored.*/
-#define AP_DESIGNER_FREESCALE    0x00e
-#define AP_DESIGNER_TEXAS        0x017
-#define AP_DESIGNER_ATMEL        0x01f
-#define AP_DESIGNER_STM          0x020
-/* CPU2 for STM32W(L|B) uses ARM JEDEC continuation (4) and
- * not STM ARM JEDEC continuation (0) as for CPU1.
- * See RM0453
- * https://www.st.com/resource/en/reference_manual/rm0453-stm32wl5x-advanced-armbased-32bit-mcus-with-subghz-radio-solution-stmicroelectronics.pdf :
- * 38.8.2 CPU1 ROM CoreSight peripheral identity register 4 (ROM_PIDR4)
- * vs
- * 38.13.2 CPU2 ROM1 CoreSight peripheral identity register 4 (C2ROM1_PIDR4)
- */
-#define AP_DESIGNER_STM32WX      0x420
-#define AP_DESIGNER_CYPRESS      0x034
-#define AP_DESIGNER_INFINEON     0x041
-#define AP_DESIGNER_NORDIC       0x244
-#define AP_DESIGNER_ARM          0x43b
-/*LPC845 with designer 501. Strange!? */
-#define AP_DESIGNER_SPECULAR     0x501
-#define AP_DESIGNER_CS           0x555
-#define AP_DESIGNER_ENERGY_MICRO 0x673
-#define AP_DESIGNER_GIGADEVICE   0x751
-#define AP_DESIGNER_RASPBERRY    0x927
+#define ADIV5_AP_CFG  ADIV5_AP_REG(0xF4)
+#define ADIV5_AP_BASE ADIV5_AP_REG(0xF8)
+#define ADIV5_AP_IDR  ADIV5_AP_REG(0xFC)
 
 /* AP Control and Status Word (CSW) */
-#define ADIV5_AP_CSW_DBGSWENABLE	(1u << 31)
+#define ADIV5_AP_CSW_DBGSWENABLE (1u << 31)
```

Comment:
clang-tidy emits warnings when the U is lower case - please convert to upper case as you're doing the big reformat on these files.